### ISSUE TITLE
可以得到mtu值吗 (Can I get the mtu value?)

### ISSUE THREAD

**Issue by mengsiyitiaojie** [point-source/dart_ping#13]
可以得到mtu值吗

--translation--

Can I get the mtu value?

**Comment by point-source** [comment]
You want the mtu of the ping request or response or of the network path?

For the request or response, it would just be the ping packet size added to the header. I can possibly expose a packet size argument so you can specify this but I'm not sure how to figure out what the header is since that depends somewhat on your network connection details.

As for calculating the total mtu of the network path, that's out of scope for this tool but if I add the packet size argument, then you should be able to create a tool based on dart_ping which will allow you to calculate mtu.

In short, I need a bit more detail about what you are trying to do.

---translation---

您是否需要ping请求或响应的MTU或网络路径的MTU？

对于请求或响应，它只是添加到标头中的ping数据包大小。我可能会公开一个数据包大小参数，以便您可以指定此参数，但我不确定如何确定标头是什么，因为这在某种程度上取决于您的网络连接详细信息。

至于计算网络路径的总mtu，这超出了此工具的范围，但是如果我添加了数据包大小参数，则您应该能够基于dart_ping创建一个工具，该工具将允许您计算mtu。

简而言之，我需要更多有关您要执行的操作的详细信息。

**Comment by point-source** [comment]
Closing due to inactivity / lack of clarity regarding the issue
